spring-batch-docs/asciidoc/retry.adoc

```diff
@@ -210,70 +210,73 @@ Be very careful with the implementation of
         `Object.equals()` and `Object.hashCode()` in the
         key returned by `RetryState`. The best advice is
         to use a business key to identify the items. In the case of a JMS
-        message the message ID can be used.
+        message, the message ID can be used.
 ====
 
 
-When the retry is exhausted there is also the option to handle the
+When the retry is exhausted, there is also the option to handle the
       failed item in a different way, instead of calling the
-      `RetryCallback` (which is presumed now to be likely
+      `RetryCallback` (which is now presumed to be likely
       to fail). Just like in the stateless case, this option is provided by
       the `RecoveryCallback`, which can be provided by
       passing it in to the `execute` method of
       `RetryOperations`.
 
 The decision to retry or not is actually delegated to a regular
       `RetryPolicy`, so the usual concerns about limits
-      and timeouts can be injected there (see below).
+      and timeouts can be injected there (described later in this chapter).
+
+// TODO This section has no example. It could use one (or more) to show how to inject
```

Comment:
We can remove this comment.  I opened story https://jira.spring.io/browse/BATCH-2645 to cover this issue.

Reply:
Thank you. :)